Formula/c/cataclysm.rb

```diff
@@ -39,6 +39,7 @@ class Cataclysm < Formula
 
   on_macos do
     depends_on "gettext"
+    depends_on "freetype" if build.head?
```

Comment:
`build.head?` does not work outside of `install` methods. Instead, you need to use a `head do` block.

Reply:
Removed the head condition since the make is always in the head implicitly